src/rpdk/core/contract/suite/handler_commons.py

```diff
@@ -163,32 +163,7 @@ def test_input_equals_output(resource_client, input_model, output_model):
     pruned_output_model = prune_properties_if_not_exist_in_path(
         pruned_output_model, pruned_input_model, resource_client.create_only_paths
     )
-
```

Comment:
I cant attach a comment to the code above but I wanna make sure that `input_model` is the provided template and not the resource model from previous action, isnt it?

Reply:
could you explain what you mean by ```resource model from previous action``` are you referring to response model of an action ? because when we compare for Update we do call the same function. This is an example: https://github.com/aws-cloudformation/cloudformation-cli/blob/master/src/rpdk/core/contract/suite/handler_update.py#L33

Reply:
gotcha, yea I meant to make sure it's not a response from the handler we comparing